src/commands/go-architect.yaml

```diff
@@ -0,0 +1,19 @@
+parameters:
```

Comment:
I'd like to rename it to have a `-legacy` suffix. At some point we should delete all those legacy jobs.

Reply:
Makes sense.

Reply:
Done.